doc/api/async_hooks.md

```diff
@@ -460,6 +460,57 @@ init for PROMISE with id 6, trigger id: 5  # the Promise returned by then()
   after 6
 ```
 
+#### async_hooks.currentResource()
```

Comment:
Should it be `executionAsyncResource()` here and in all places below?

Reply:
Yep, thanks for the reminder. 👍